gradle.properties

```diff
@@ -1,54 +1,23 @@
+# Gradle
 org.gradle.jvmargs=-Xmx4096m -Dfile.encoding=UTF-8
+
+# Android
 android.useAndroidX=true
-android.enableJetifier=true
-kotlin.code.style=official
+android.defaults.buildfeatures.buildConfig=false
+android.library.defaults.buildfeatures.androidresources=false
 
+# Kotlin
+kotlin.code.style=official
 kotlin.native.useEmbeddableCompilerJar=true
-# Enable kotlin/native experimental memory model
 kotlin.native.ignoreDisabledTargets=true
-kotlin.native.binary.memoryModel=experimental
 kotlin.mpp.stability.nowarn=true
 
-# Compose dependencies currently requires to be CompileOny if we want to both
-# support Android only projects and multiplatform projects.
-# based on the issue below, it seems to only have issues with compileOnly
-# when cache is enable, so, based on the currently state of Compose Multiplatform
-# the native target also does not work propertly with caching, so, any compose project
-# will require `kotlin.native.cacheKind=none` making possible to we continue using compileOnly.
-# https://youtrack.jetbrains.com/issue/KT-46377
-kotlin.native.ignoreIncorrectDependencies=true
-
-android.defaults.buildfeatures.buildconfig = false
-
+# Compose multiplatform
 org.jetbrains.compose.experimental.macos.enabled=true
 org.jetbrains.compose.experimental.jscanvas.enabled=true
 org.jetbrains.compose.experimental.wasm.enabled=true
 
+# Atomicfu
 kotlinx.atomicfu.enableJvmIrTransformation=true
 kotlinx.atomicfu.enableNativeIrTransformation=true
 kotlinx.atomicfu.enableJsIrTransformation=true
-
-kotlin.mpp.androidSourceSetLayoutVersion=2
-
-# Maven
-SONATYPE_HOST=S01
-SONATYPE_AUTOMATIC_RELEASE=true
-RELEASE_SIGNING_ENABLED=true
-
-GROUP=cafe.adriel.voyager
-
-POM_DESCRIPTION=A pragmatic navigation library for Jetpack Compose
-POM_INCEPTION_YEAR=2021
-POM_URL=https://github.com/adrielcafe/voyager
-
-POM_LICENCE_NAME=The MIT License
-POM_LICENCE_URL=https://opensource.org/licenses/MIT
-POM_LICENCE_DIST=repo
-
-POM_SCM_URL=https://github.com/adrielcafe/voyager
-POM_SCM_CONNECTION=scm:git:ssh://git@github.com/adrielcafe/voyager.git
-POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/adrielcafe/voyager.git
-
-POM_DEVELOPER_ID=adrielcafe
-POM_DEVELOPER_NAME=Adriel Cafe
-POM_DEVELOPER_URL=https://github.com/adrielcafe/
```

Comment:
Why have you move this to code? The plugin does not support gradle properties anymore?